packages/shared/src/lib/utils/config.ts

```diff
@@ -1,6 +1,7 @@
 import { IConfig } from "../types";
 
 export const config: IConfig = {
+  builderId: "request-team",
```

Comment:
Please change to empty string.
```suggestion
  builderId: "",
```